Assets/Scripts/StandardSamples/UI/Storage/Player/UIPlayerDataStorageMenu.cs

```diff
@@ -307,15 +307,17 @@ public override void Show()
         {
             base.Show();
             UpdateFileListUI();
-            PlayerDataStorageManager.AddNotifyFileListUpdated(UpdateFileListUI);
-            PlayerDataStorageManager.OnLoggedIn();
```

Comment:
Double checking, are we intentionally no longer calling `OnLoggedIn` and `OnLoggedOut` for `Show` and `Hide`?

Reply:
This is correct. The event of the scene being shown or hidden is independent from the event of the user logging in. It's possible for instance for a scene that is hidden to still take action when logging in takes place (by updating locally cached information that will subsequently be displayed when and if the scene _is_ shown). Likewise, even hidden scenes should take action when logging out happens - because they still have data that needs clearing.